uaa/src/test/java/org/cloudfoundry/identity/uaa/login/LoginMockMvcTests.java

```diff
@@ -2665,10 +2668,12 @@ void emailPageIdpDiscoveryEnabled_SelfServiceLinksDisabled(
     ) throws Exception {
         IdentityZoneConfiguration config = new IdentityZoneConfiguration();
         config.setIdpDiscoveryEnabled(true);
-        config.setLinks(new Links().setSelfService(new Links.SelfService().setSelfServiceLinksEnabled(false)));
+        config.setLinks(new Links().setSelfService(new Links.SelfService().setSelfServiceCreateAccountEnabled(false)));
```

Comment:
We have a separate story US554095 to check assertion for both emailPageIdpDiscoveryEnabled_SelfServiceLinksDisabled() passwordPageIdpDiscoveryEnabled_SelfServiceLinksDisabled() - I believe the tests will pass regardless of either self-service is disabled or enabled.  Not sure what's better, change these tests to use new flags or defer changing these tests until we work on the US554095...